package.json

```diff
@@ -22,6 +22,7 @@
     "@oclif/config": "^1",
     "@oclif/plugin-help": "^2",
     "@oclif/plugin-not-found": "^1.2",
+    "@types/deepmerge": "^2.2.0",
```

Comment:
I've never understood this -- do these need to be dependencies of our package? Since agent is distributed as a compiled package it feels weird that everyone who installs our package also gets a bunch of types packages. too.

Reply:
This is a good SO for this: https://stackoverflow.com/a/46011417

Reply:
Shifted all the types to dev deps in 65f535220ae7d0dc4b299524026b862048c02333